lnwire/query_short_chan_ids.go

```diff
@@ -67,6 +80,12 @@ type QueryShortChanIDs struct {
 
 	// ShortChanIDs is a slice of decoded short channel ID's.
 	ShortChanIDs []ShortChannelID
+
+	// noSort indicates whether or not to sort the short channel ids before
+	// writing them out.
+	//
+	// NOTE: This should only be used during testing.
+	noSort bool
```

Comment:
Any reason to add this? They should always be sorted so this is only used for testing unused behavior.

Reply:
we always sort, but this tests that we behave as expected if non-lnd peers don't sort. mainly this was the simplest way of passing the bool internally without modifying `Encode` or `Decode`, or having to duplicate the encoding logic w/o sorting just for testing